rosdep/python.yaml

```diff
@@ -6955,6 +6955,10 @@ python3-kitchen:
   ubuntu:
     '*': [python3-kitchen]
     xenial: null
+python3-kivy-pip:
```

Comment:
This is also packaged upstream so pip rules should not be used for it. 

https://packages.ubuntu.com/jammy/python-kivy-examples